src/scss/modifiers/_inverse.scss

```diff
@@ -29,5 +30,50 @@
 				border-color: _oFormsGet('toggle-inverse-selected');
 			}
 		}
+
+		.o-forms-input--invalid {
+			*:invalid:not(:disabled) {
+				color: _oFormsGet('invalid-base-color-inverse');
+				border-color: _oFormsGet('invalid-base-border-inverse');
+
+				+ .o-forms-input__label:before,
+				+ .o-forms-input__label:after {
+					border-color: _oFormsGet('invalid-base-border-inverse');
+				}
+			}
+
+			.o-forms-input__error {
+				color: _oFormsGet('invalid-base-inverse');
+			}
+		}
+
+		@if $radio {
```

Comment:
`$radio` represents radio-round, you want radio-box! 😅

Reply:
fixed in [`55b4eaf` (#376)](https://github.com/Financial-Times/o-forms/pull/376/commits/55b4eafdcd7464f0da98eff89151859afac7c930) 👍